.github/workflows/e2e-tests.yml

```diff
@@ -134,11 +139,10 @@ jobs:
           password: ${{ secrets.DOCKERHUB_TOKEN }}
     steps:
       - uses: actions/checkout@v3
-      - name: Set conditional ENVs
+      - name: Record runs using Deploysentinel except for the release branch
+        if: ${{ github.ref == 'refs/heads/master' || !(startsWith(github.event.pull_request.base.ref, 'release')) }}
```

Comment:
Basically, ignore anything related to the release branch:
1. direct pushes to the release branch
2. PRs against the release branch